README.md

```diff
@@ -498,3 +498,51 @@ Some of the options include:
 * `members` - automatically document the members in that Python module.
 * `private-members` - include private functions and methods.
 * `show-inheritance` - show the class inheritance.
+
+# Opentelemetry Instrumentation
+
+IIB is now implemented with Opentelemetry python instrumentation.
+This is done in order to collect detailed information about the behavior and performance of IIB.
+By using a combination of automatic and manual instrumentation, it is possible to capture all relevant information about the application's behavior.
+
+## Environment setup
```

Comment:
I am wondering if we should have some general OpenTelemetry  setup within our dev-env to be able to test things without any external software/api. (somethign like opentelemetry container). Since we have everything in devenv.

_# Maybe my question is wrong - I need to study opentelemetry little bit more_

Reply:
Good idea though and I would suggest taking creating a JIRA to address that within your team

Reply:
Same comment was raised here: https://github.com/release-engineering/iib/pull/508#discussion_r1167246200